client/layout/index.jsx

```diff
@@ -342,6 +342,9 @@ class Layout extends Component {
 				{ config.isEnabled( 'legal-updates-banner' ) && (
 					<AsyncLoad require="calypso/blocks/legal-updates-banner" placeholder={ null } />
 				) }
+				{ config.isEnabled( 'yolo/command-pallette' ) && ! isJetpackCloud() && (
```

Comment:
My plan when releasing the feature is removing the feature flag ~`config.isEnabled( 'yolo/command-pallette' )`~ and leaving only the `! isJetpackCloud()` check.

Reply:
I understand, but my comment still applies. Instead of having the `isJetpackCloud()` check, we can still rely on feature flag checks. If we don't enable the feature flag on the Jetpack Cloud environments, we'll not need the extra `isJetpackCloud()` check. After all, that's what that check does anyway:

```
const jetpackCloudEnvironments = [
	'jetpack-cloud-development',
	'jetpack-cloud-stage',
	'jetpack-cloud-horizon',
	'jetpack-cloud-production',
];

const isJetpackCloud = (): jetpackCloudEnvironments.includes( config( 'env_id' ) );
```

Reply:
In other words, feature flags aren't necessarily removed when a feature is released to production. Feature flags can remain for the time being when we want to intentionally enable/disable a feature based on the Calypso environment.

Reply:
Thanks for clarifying !